Source/Core/Common/CMakeLists.txt

```diff
@@ -65,4 +65,18 @@ if(NOT APPLE AND NOT ANDROID)
 	set(LIBS ${LIBS} rt)
 endif()
 
+if(VTUNE)
```

Comment:
It would be nice to follow our cmake setup rather than sticking things in "arbitrary" locations.